Nitrox.Test/Patcher/Patches/Dynamic/SpawnOnKill_OnKill_PatchTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using FluentAssertions;
+using HarmonyLib;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NitroxTest.Patcher;
+using static NitroxPatcher.Patches.Dynamic.SpawnOnKill_OnKill_Patch;
+
+namespace Nitrox.Test.Patcher.Patches.Dynamic;
```

Comment:
```suggestion
namespace NitroxPatcher.Patches.Dynamic;
```
As explained by Measurity in #2017, we now have a defined namespace rule, which you'll also apply when making the test for BreakableResource's transpiler